plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsMysqlFlywayMigration.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.resourcegroups.db;
+
+import org.jdbi.v3.core.Handle;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testng.annotations.Test;
+
+public class TestDbResourceGroupsMysqlFlywayMigration
+        extends TestDbResourceGroupsFlywayMigration
+{
+    @Override
+    protected final JdbcDatabaseContainer<?> startContainer()
+    {
+        JdbcDatabaseContainer<?> container = new MySQLContainer<>("mysql:8.0.12");
```

Comment:
@ebyhr @wendigo What do you think about starting to read image versions from a single properties file (or public constants) to ensure consistency in the versions being tested (easier to bump min and max tested versions that way).

@posulliv No change requested.

Reply:
An alternative is to make versions injectable either from environment or system properties. That has the benefit that it's easier to test arbitrary versions via a matrix.